lyricsgenius/genius.py

```diff
@@ -524,12 +520,6 @@ def find_artist_id(search_term):
             return None
 
         artist_info = self.artist(artist_id)['artist']
-        found_name = artist_info['name']
-        if found_name != artist_name and allow_name_change:
-            if self.verbose:
-                print("Changing artist name to '{a}'".format(
```

Comment:
The `artist_name` variable is indeed not used anymore, but let's keep the print statement to avoid surprises.

Reply:
Fair point, a bash script may be parsing the standard output to do things.

That then falls too under the changes that can only be done on a new major release.